pkg/agent/cniserver/pod_configuration.go

```diff
@@ -178,7 +167,7 @@ func ParseOVSPortInterfaceConfig(portData *ovsconfig.OVSPortData, portConfig *in
 
 	containerMAC, err := net.ParseMAC(portData.ExternalIDs[ovsExternalIDMAC])
 	if err != nil {
-		klog.Errorf("Failed to parse MAC address from OVS external config %s: %v",
+		klog.Warningf("Failed to parse MAC address from OVS external config %s: %v",
```

Comment:
based on the code above (`createContainerLink`), won't we get a warning for every Pod? If yes, could we use a dummy MAC address instead of `nil` in `createContainerLink` to avoid the warning / error?

Reply:
Done, set a dummy MAC.